src/sentry/api/serializers/models/projectcodeowners.py

```diff
@@ -38,8 +39,8 @@ def get_attrs(self, item_list, user, **kwargs):
                 organization_id=item.repository_project_path_config.organization_id,
             )
             codeowners_url = "unknown"
-            if item.repository_project_path_config.organization_integration_id and isinstance(
-                install, RepositoryMixin
+            if item.repository_project_path_config.organization_integration_id and (
+                isinstance(install, RepositoryMixin) or isinstance(install, RepositoryIntegration)
```

Comment:
Nit: this (and all similar `isinstance` checks) can be written as `isinstance(install, (RepositoryMixin, RepositoryIntegration))`. (Assuming we don't just delete `RepositoryMixin`.)